src/sentry/notifications/notificationcontroller.py

```diff
@@ -58,14 +61,24 @@ def __init__(
         organization_id: int | None = None,
         type: NotificationSettingEnum | None = None,
         provider: ExternalProviderEnum | None = None,
+        organization: Organization | None = None,
```

Comment:
is there a better way to get organization without needing to have this passed in? would need to update every single instantiation of NotificationController

Reply:
@isabellaenriquez we already pass in `organization_id` and `project_ids`. Should be able to look up the org from one of those.

Reply:
when I tried to query for organization using organization_id I got this error: `sentry.silo.base.SiloLimit.AvailabilityError: Called `Organization.get_queryset` on server in CONTROL mode. Organization is available only in: MONOLITH, REGION`. Is there a way to enable that here or another way to get organization using organization ID?